javascript/chronometer.js

```diff
@@ -1,26 +1,57 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+
+    this.currentTime = 0;
+    this.intervalId = 0;
+
   }
   startClick(callback) {
     // ... your code goes here
+
+    this.intervalId = setInterval(() => {
+      this.currentTime ++ 
+      
+      callback();
```

Comment:
Es una solución muy buena 🔝 
Como consejo, Jasmine al pasar los test no os está pasando ningún `callback`, por tanto tenéis dos test que no os pasarán. Para solucionar eso podemos añadir una condicional que comprueba que `callback` es una función y si es así ejecutala:
```
if (typeof callback === 'function') {
  callback();
}
```